### DISCUSSION TITLE
Optional items arrangement: Add sections with horizontal items srolling

### DISCUSSION THREAD

**Discussion by esQmo** [trakt/showly#430] [Ideas / Feature Requests]
Could Showly2.0 have sections with horizontal items scrolling in its main pages? I mean movies or tv shows can be sectionned and their contents grouped like Sci, Thrillers, Drama... like we see in most movie apps. I'm thinking about it as an optional feature an user can choose to enable or not.

Of course the way it is now is good too, no problem with that.

**Comment by michaldrabik** [comment]
Hey, I know that almost every other app uses horizontal lists in their "Discover" sections but I wanted to create something different. So far people seem to like it. Perhaps I could think of some other way of categorizing discover content by genres / networks that would match the current feel in a better way.

**Reply by esQmo** [reply]
Hello, thanks. I think we're going to like it. We like Showly the way it is now as well.
